geode-core/src/main/java/org/apache/geode/internal/i18n/LocalizedStrings.java

```diff
@@ -7713,6 +7713,8 @@
       new StringId(6664, "{0}: Providing synchronization event for key={1}; timestamp={2}: {3}");
   public static final StringId AbstractGatewaySender_ENQUEUEING_SYNCHRONIZATION_EVENT =
       new StringId(6665, "{0}: Enqueueing synchronization event: {1}");
+  public static final StringId GatewayReceiver_ALREADY_EXISTS =
+      new StringId(6665, "A Gateway Receiver already exists on this member.");
```

Comment:
This StringId probably needs to be bumped to 6666 or something that isn't already being used.

Reply:
Thanks @jhuynh1,
I'll directly remove the constant from the `LocalizedStrings` class, missed that discussion in the dev list.